Lib/email/_header_value_parser.py

```diff
@@ -1101,13 +1101,13 @@ def get_unstructured(value):
             token, value = get_fws(value)
             unstructured.append(token)
             continue
+        encoded_format_invalid = False
```

Comment:
Can I suggest `invalid_ew` instead for this variable? (`ew` a.k.a encoded-word) is more commonly used and is more explanatory than encoded_format.

Reply:
Good idea. I renamed it to `valid_ew` instead.